<!--
  Used for visualising the turtlebot while building a map or navigating with the ros navistack.
 -->
<launch>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find turtlebot_rviz_launchers)/rviz/navigation.rviz">
    <remap from="/move_base_simple/goal" to="/rviz_navigation_goal"/>
  </node>
  <node name="experiment_manager" pkg="hri_planner" type="experiment_manager.py" output="screen">
    <param name="protocol_file" value="$(find hri_planner)/resources/exp_protocols/protocol.txt"/>
  </node>
</launch>
